demos/2019_RHTE/labs/8.md

```diff
@@ -92,17 +92,14 @@ spec:
 
   # [!] Change namespaces to adjust which OpenShift namespaces should be migrated from source to destination cluster
   namespaces:
-  -
+  namespaces:
```

Comment:
Are you sure you want to see 'namespaces' twice?

namespaces:
namespaces:


Maybe this is a typo and you only want it once?

Reply:
typo...thanks!